src/libcore/ptr/mod.rs

```diff
@@ -839,6 +853,20 @@ pub unsafe fn write<T>(dst: *mut T, src: T) {
 ///
 /// Accessing unaligned fields directly with e.g. `packed.unaligned` is safe however.
 // FIXME: Update docs based on outcome of RFC #2582 and friends.
+///
+/// # Examples
+///
+/// Write an usize value to a byte buffer:
+///
+/// ```
+/// fn write_usize(x: &mut [u8], val: usize) {
+///   assert!(x.len() >= mem::size_of::<usize>());
```

Comment:
Could you change the indentation here? I believe Rust prefer 4-spaces indentation.